catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -0,0 +1,326 @@
+                opsSecurity,
+                opsMetacard,
+                opsCatStore);
+
+        // set up deleteRequest
+        MetacardImpl metacard = new MetacardImpl();
+        ArrayList<Metacard> metacardList = new ArrayList<>();
+        metacard.setId("Bob");
+        metacard.setTitle("Bob's Title");
+        metacardList.add(metacard);
+
+        metacard = new MetacardImpl();
+        metacard.setId("Bobbert");
+        metacard.setTitle("Bobbert's Title");
+        metacardList.add(metacard);
+
+        deleteRequest = new DeleteRequestImpl(metacard.getId());
+        deleteResponse = new DeleteResponseImpl(deleteRequest, new HashMap(), metacardList);
+
+        // set up blacklist
+        fanoutTagBlacklist = new ArrayList<>();
+        fanoutTagBlacklist.add("");
+
+        opsSecuritySupport = mock(OperationsSecuritySupport.class);
+
+        opsMetacardSupport = mock(OperationsMetacardSupport.class);
+
+        opsCatStoreSupport = mock(OperationsCatalogStoreSupport.class);
+
+        remoteDeleteOperations = new RemoteDeleteOperations(frameworkProperties,
+                opsMetacardSupport,
+                opsCatStoreSupport);
+
+    }
+
+    @Test
+    public void testIsNotCatalogStoreRequest() throws Exception {
+
+        when(opsCatStoreSupport.isCatalogStoreRequest(deleteRequest)).thenReturn(false);
+        remoteDeleteOperations.setOpsCatStoreSupport(opsCatStoreSupport);
+
+        DeleteResponse resultDeleteResponse = remoteDeleteOperations.performRemoteDelete(
+                deleteRequest,
+                deleteResponse);
+
+        assertThat("Assert return of identical deleteResponse",
+                resultDeleteResponse.getDeletedMetacards() == deleteResponse.getDeletedMetacards());
+        verify(opsCatStoreSupport).isCatalogStoreRequest(deleteRequest);
+
+    }
+
+    @Test
+    public void testIsCatalogStoreRequest() throws Exception {
+
+        when(opsCatStoreSupport.isCatalogStoreRequest(deleteRequest)).thenReturn(true);
+        remoteDeleteOperations.setOpsCatStoreSupport(opsCatStoreSupport);
+
+        DeleteResponse resultDeleteResponse = remoteDeleteOperations.performRemoteDelete(
+                deleteRequest,
+                deleteResponse);
+
+        assertThat("Assert return of identical deleteResponse",
+                resultDeleteResponse.getDeletedMetacards() == deleteResponse.getDeletedMetacards());
+        verify(opsCatStoreSupport).isCatalogStoreRequest(deleteRequest);
+    }
+
+    @Test
+    public void testDeleteResponseNull() throws Exception {
+
+        when(opsCatStoreSupport.isCatalogStoreRequest(deleteRequest)).thenReturn(true);
+        remoteDeleteOperations.setOpsCatStoreSupport(opsCatStoreSupport);
+
+        deleteResponse = null;
+
+        DeleteResponse resultDeleteResponse = remoteDeleteOperations.performRemoteDelete(
+                deleteRequest,
+                deleteResponse);
+
+        assertThat("Null request should return an empty response",
```

Comment:
JUnit's `Assert.assertThat()` method allows you to write more descriptive tests since you can use `Matcher`s rather than writing your own conditionals. For example, you could write `assertThat("Null request did not return an empty response", resultDeleteResponse.getDeletedMetacards(), is(empty()));`